src/utilities/setup.py

```diff
@@ -8,6 +8,7 @@
 NUM_GAMES = 3
 # certainties from costum vision lower than this -> haswon=False
 CERTAINTY_THRESHOLD = 0.7
+SAVE_CERTAINTY = 0.3
```

Comment:
Can you add a comment which documents the purpose of this variable?

Reply:
👌